### PR TITLE
Adjust CR documentation to the new guidelines

### PR DESCRIPTION
**Pull request by klaudiagrz**

### CODE REVIEW

File: docs/api-gateway/docs/011-api-custom-resource.md [hunk 1]
```diff
@@ -35,8 +35,6 @@ This table analyses the elements of the sample CR and the information it contain
 
 | Field   |      Mandatory?      |  Description |
 |:----------:|:-------------:|:------|
-| **apiVersion** |  **YES** | Defined basing on the **group** and **version** fields of the CRD **spec** section. |
```
Comment (tomekpapiernik) [inline docs/api-gateway/docs/011-api-custom-resource.md]: Why delete these fields? They are a part of every CR.

Comment (klaudiagrz) [reply docs/api-gateway/docs/011-api-custom-resource.md]: Yup, that's why @derberg decided not to include them in the table as they are obvious.

Comment (derberg) [reply docs/api-gateway/docs/011-api-custom-resource.md]: exactly, these are basic fields that are part of every resource, not only CRDs

Comment (tomekpapiernik) [reply docs/api-gateway/docs/011-api-custom-resource.md]: okaaaay

**Pull request merged**